website/image_drop.html

```diff
@@ -0,0 +1,197 @@
+<!doctype html>
```

Comment:
Since we have an "About" and "Contact" page by themselves now, makes sense to remove them from here. The white background here actually works much better than the green on the front page.

I would leave in the top 5 retailers part for now it looks cool.